commons/src/test/scala/dagr/commons/util/StringUtilTest.scala

```diff
@@ -21,36 +21,48 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
-package dagr.core.util
+package dagr.commons.util
 
 /**
   * Tests for StringUtil
   */
 class StringUtilTest extends UnitSpec {
-  "StringUtil.camelToGnu" should "correctly handle simple multi-word variable names" in {
-    StringUtil.camelToGnu("fooBar") should be ("foo-bar")
-    StringUtil.camelToGnu("oneTwoThreeFourFiveSixSeven") should be ("one-two-three-four-five-six-seven")
-  }
 
-  it should "not modify single word variable names" in {
-    StringUtil.camelToGnu("foo") should be ("foo")
+  "StringUtil.wrapString" should "wrap a string with a prefix and suffix" in {
+    StringUtil.wrapString("pre-", "body", "-suffix") shouldBe "pre-body-suffix"
```

Comment:
Since we also do wrapping in the sense of word-wrap, perhaps this method should be called `StringUtil.enclose()`?